packages/suite/src/components/suite/modals/TransactionDetail/components/ChangeFee/index.tsx

```diff
@@ -118,18 +118,19 @@ const ChangeFee = (props: Props & { showChained: () => void }) => {
                                 <Rate>{feeRate}</Rate>
                                 <Amount>
                                     <StyledCryptoAmount>
-                                        <FormattedCryptoAmount value={tx.fee} symbol={tx.symbol} />
+                                        <FormattedCryptoAmount
+                                            disableHiddenPlaceholder
```

Comment:
Why do we disable the blurring here? From the commit message I got the impression the goal is the opposite: to enable it. I can see we do that in `DecreasedOutputs/index.tsx`, but the message doesn't mention we should disable the blur here.

Reply:
because it's not a sensitive information. it's like in send form - it's not blurred there since you are working with this amounts

Reply:
in other hand output amount or output address ARE sensitive, you can find this transaction in blockchain if you do a photo/screenshot of it

Reply:
I was asking because when I review a change-set I check that it implements what it is said to implement in the commit message/PR descirption/issue. Here, it implements something *above* the commit message. It would be cool to mention this change in the commit message as well. .) 

Anyway, I'm ok with both outcomes (blurring or not) so RBY as you wish and merge. .)